acceptance/openstack/blockstorage/v3/volumetypes_test.go

```diff
@@ -45,7 +45,7 @@ func TestVolumeTypesList(t *testing.T) {
 	}
 }
 
-func TestVolumeTypesCreate(t *testing.T) {
+func TestVolumeTypesCreateDestroy(t *testing.T) {
```

Comment:
Assuming you're going to do the Update test here, too, please change to `TestVolumeTypesCRUD` in the next PR.